rust/utils/run-locally/src/invariants.rs

```diff
@@ -63,19 +63,20 @@ pub fn termination_invariants_met(
     .sum::<u32>();
     // TestSendReceiver randomly breaks gas payments up into
     // two. So we expect at least as many gas payments as messages.
-    if gas_payment_events_count < total_messages_expected {
-        log!(
-            "Relayer has {} gas payment events, expected at least {}",
-            gas_payment_events_count,
-            total_messages_expected
-        );
-        return Ok(false);
-    }
+    // TODO: fix this once eth gas payments are introduced
+    // if gas_payment_events_count < total_messages_expected {
```

Comment:
I assume we'll just revisit this later?

Reply:
I think this was commented out because double insertions wouldn't work without setting the gas to zero, which made this condition fail